### PR TITLE
TanksButNoTanks

### PR DESCRIPTION
**Pull request by linuxgurugamer**
created by new automated script
hand verified
Closes #4391

Added MM depends stanza manually, it wasn't in the zip file

### CODE REVIEW

File: NetKAN/TanksButNoTanks.netkan [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "license": "GPL-3.0",
+  "identifier": "TanksButNoTanks",
+  "x_via": "Automated SpaceDock CKAN submission",
+  "$kref": "#/ckan/spacedock/842",
+  "depends": [
+    {
+      "name": "ContractConfigurator"
```
Comment (politas) [inline NetKAN/TanksButNoTanks.netkan]: Shouldn't this be ModuleManager?

**Comment by linuxgurugamer** [comment]
My bad, fixed

**Comment by politas** [comment]
LGTM. Merging with thanks!

**Comment by politas** [comment]
Whoops!

**Pull request merged**